src/client/thirdParty/Inkeep.tsx

```diff
@@ -26,17 +33,39 @@ type InkeepSharedSettings = {
   modalSettings: InkeepModalSettings;
 };
 
-const useInkeepSettings = (): InkeepSharedSettings => {
+export const createCustomAnalyticsCallback = (
+  getVisitorData: (getDataOptions?: GetDataOptions<boolean>) => Promise<VisitorData<boolean>>,
+  location: LocationSensorState,
+) => {
+  return async (event: any) => {
+    if (event.eventName === GET_HELP_OPTIONS_CLICKED) {
+      const { visitorId } = await getVisitorData({ ignoreCache: true });
+
+      const { name } = event.properties;
+      const pagePath = location.pathname || '';
```

Comment:
Why not just use `document.location.pathname`  and then you don't need to pass `location` through all those functions?